core/services/ocr2/delegate.go

```diff
@@ -697,6 +703,11 @@ func (d *Delegate) newServicesGenericPlugin(
 		}
 		srvs = append(srvs, job.NewServiceAdapter(oracle))
 
+		if externalPeerWrapper != nil {
+			syncer := capabilities.NewSyncer(externalPeerWrapper.GetPeer(), capabilitiesRegistry, false, lggr)
```

Comment:
@bolekk Not sure I understand why we're instantiating the syncer here. This means that for any generic plugin we'll instantiate a new syncer which doesn't seem quite right.

IMO the syncer should run outside of the job spawner as a core part of the application.